sphinx/themes/basic/static/searchtools.js

```diff
@@ -39,6 +39,7 @@ if (typeof Scorer === "undefined") {
     objPrioDefault: 0,
 
     // query found in title
+    mainTitle: 16,
```

Comment:
I've been thinking about this a bit-- technically this breaks backwards compatibility if someone defined a "custom" scorer. I think we might be better off doing something like:

```js
if (typeof Scorer === "undefined") {
  Scorer = {};
}
Scorer = {
   ...Scorer,
   mainTitle: 16,
   ...
}
```

That way we can add new properties to the class and if the user over-rode it we'd fall back to the new default values.

Reply:
Hm.  That is true; but could that breakage be avoided by setting the original `Scorer` class as the `__proto__` (prototype) for the custom class?  Inheritance-wise, I _think_ that would be the safe way to overload the values -- and perhaps a valid migration path for any sites that had simply defined their own without using inheritance?